tikv_metrics_proxy/README.md

```diff
@@ -0,0 +1,49 @@
+TiKV metrics proxy
+------
+
+TiKV metrics proxy is a tool for exposing Prometheus metrics of TiKV server. The `tikv_metrics_proxy` exposes metrics via HTTP on the `/metrics` endpoint. 
+
+#### Build
+
+```
+make tikv_metrics_proxy
+```
+
+#### Running
+
+```
+$ ./tikv_metrics_proxy -h
+usage: tikv_metrics_proxy [<flags>]
+
+Flags:
+  -h, --help              Show context-sensitive help (also try --help-long and --help-man).
+      --web.listen-address=":9600"
+                          Address on which to expose metrics and web interface.
+      --web.telemetry-path="/metrics"
```

Comment:
`/metrics` is the common path for Prometheus, I think we don't need to provide config for it

Reply:
Yes. Providing `web.telemetry-path` flag is common way for prometheus exporters, like node_exporter and influxdb_exporter. We could hide it in ansible or docker deployment.